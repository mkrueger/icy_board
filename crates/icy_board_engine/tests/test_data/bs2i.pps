PrintLn BS2I("1231"), ",", BS2I(I2BS(1234))
PrintLn BD2I("12345678")