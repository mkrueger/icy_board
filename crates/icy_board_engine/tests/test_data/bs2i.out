91748,1234
12006965
